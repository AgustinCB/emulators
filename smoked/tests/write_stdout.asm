.data
INTEGER 1
INTEGER 3
STRING "HOLA PAPA
"
INTEGER 10
.code
CONSTANT 4
CONSTANT 3
CONSTANT 1
CONSTANT 2
CONSTANT 1
SYSCALL
