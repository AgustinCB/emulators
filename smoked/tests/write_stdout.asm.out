HOLA PAPA
0 - SimpleValue(Integer(10))
